rj_gameplay/rj_gameplay/tactic/nmark_tactic.py

```diff
@@ -35,7 +35,7 @@ def __init__(self, enemy_to_mark: rc.Robot=None):
     def __call__(
         self,
         robot: rc.Robot,
-        prev_result: Optional["RoleResult"],
+        prev_result: Optional[role.RoleResult],
```

Comment:
Error: rj_gameplay/rj_gameplay/tactic/nmark_tactic.py:38: error: Name 'RoleResult' is not defined
-> Imported static type from stp.role and replaced optional static type with actual static type